### PR TITLE
Fixed second level cache supports

### PR DESCRIPTION
**Pull request by gianarb**
Reference #431 

@eminetto I'm working to fix your bug, can I use this commit or you are interesting to update yours PR? :) Or can you help me with a review? 

Thanks! :)

### CODE REVIEW

**Comment by eminetto** [comment]
I'm glad to help with a review

**Comment by gianarb** [comment]
There are a lot of problem between doctrine/orm 2.4 and 2.5.. are you using 2.4 correct? Can you try to run your codebase with this patch? Thanks

**Comment by eminetto** [comment]
The problem happened when I upgraded from 2.5.0 to 2.5.1 Doctrine

**Comment by gianarb** [comment]
Yeah there was a problem with Version checks.. This PR resolve his bug

**Comment by gianarb** [comment]
@ocramius ping review please

2015-09-01 2:29 GMT+02:00 Elton Minetto <notifications@github.com>:

> The problem happened when I upgraded from 2.5.0 to 2.5.1 Doctrine
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/doctrine/DoctrineORMModule/pull/433#issuecomment-136536681>
> .
>



-- 
Gianluca Arbezzano
www.gianarb.it

File: src/DoctrineORMModule/Service/ConfigurationFactory.php [hunk 1]
```diff
@@ -105,7 +105,7 @@ public function createService(ServiceLocatorInterface $serviceLocator)
             }
         }
 
-        if (Version::compare('2.5.0') >= 0) {
+        if (Version::compare('2.5.0') <= 0) {
```
Comment (Ocramius) [inline src/DoctrineORMModule/Service/ConfigurationFactory.php]: Maybe simply check for `method_exists()`?

Comment (gianarb) [inline src/DoctrineORMModule/Service/ConfigurationFactory.php]: and why do you maintain this file? :D

Comment (Ocramius) [inline src/DoctrineORMModule/Service/ConfigurationFactory.php]: It's an old practice and we cannot remove it anymore.

**Comment by gianarb** [comment]
Fixed by #434

**Pull request closed without merging**